examples/resources/hpegl_vmaas_load_balancer/nsx_t_lb_pool.tf

```diff
@@ -0,0 +1,22 @@
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+resource "hpegl_vmaas_load_balancer_pool" "test_lb_pool" {
```

Comment:
I guess this is created for a load balancer, we need to take the load balacers details here as the input. https://apidocs.morpheusdata.com/#create-a-load-balancer-pool

Reply:
done